File: src/Entity/Avis.php
<?php

namespace App\Entity;

use App\Repository\AvisRepository;
use Doctrine\ORM\Mapping as ORM;

#[ORM\Entity(repositoryClass: AvisRepository::class)]
class Avis
{
    #[ORM\Id]
    #[ORM\GeneratedValue]
    #[ORM\Column]
    private ?int $id = null;

    #[ORM\Column]
    private ?float $rating = null;

    #[ORM\Column]
    private ?int $id_client = null;

    #[ORM\Column]
    private ?int $id_conducteur = null;

    public function getId(): ?int
    {
        return $this->id;
    }

    public function getRating(): ?float
    {
        return $this->rating;
    }

    public function setRating(float $rating): self
    {
        $this->rating = $rating;

        return $this;
    }

    public function getIdClient(): ?int
    {
        return $this->id_client;
    }

    public function setIdClient(int $id_client): self
    {
        $this->id_client = $id_client;

        return $this;
    }

    public function getIdConducteur(): ?int
    {
        return $this->id_conducteur;
    }

    public function setIdConducteur(int $id_conducteur): self
    {
        $this->id_conducteur = $id_conducteur;

        return $this;
    }
}

File: src/com/mycomany/entities/Avis.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package com.mycomany.entities;

/**
 *
 * @author BKHmidi
 */
public class Avis {
    private int id;
    private int rating;
    private int id_client;
    private int id_conducteur;

    public Avis(int id, int rating, int id_client, int id_conducteur) {
        this.id = id;
        this.rating = rating;
        this.id_client = id_client;
        this.id_conducteur = id_conducteur;
    }
     public Avis( int rating, int id_client, int id_conducteur) {
        this.rating = rating;
        this.id_client = id_client;
        this.id_conducteur = id_conducteur;
    }
     public Avis(){}
     public Avis(int rating){        this.rating = rating;
}

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public int getRating() {
        return rating;
    }

    public void setRating(int rating) {
        this.rating = rating;
    }

    public int getIdClient() {
        return id_client;
    }

    public void setIdClient(int id_client) {
        this.id_client = id_client;
    }

    public int getIdConducteur() {
        return id_conducteur;
    }

    public void setIdConducteur(int id_conducteur) {
        this.id_conducteur = id_conducteur;
    }
    
}
